libsignal-protocol/src/context.rs

```diff
@@ -152,7 +152,7 @@ where
     P: PreKeyStore + 'static,
     K: SignedPreKeyStore + 'static,
     S: SessionStore + 'static,
-    I: IdentityKeyStore + 'static,
+    I: IdentityKeyStore<'static> + 'static,
```

Comment:
i just said, if it has to be `'static` so why not making the lifetime for it a `'static` too :smile:

Reply:
Haha, usually when `rustc` tells me to add `'static` somewhere I interpret it as my code asking me to look back and think about how my data/ownership is being managed.

It's not wrong, but usually there's a better way than throwing `'static`s around until everything compiles... Especially if we want to provide users with a logically sound "safe" interface to `libsignal-protocol-c` :stuck_out_tongue_winking_eye: